pkgs/servers/resgate/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+}:
+
+buildGoModule rec {
+  pname = "resgate";
+  version = "1.7.5";
+
+  src = fetchFromGitHub {
+    owner = "resgateio";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-CAK2BjHa/l4cAWUKL0hGjqKi/Cdg+/K/MlnDreB69YE=";
+  };
+
+  vendorHash = "sha256-6uLCZvvQ8lRug6TlavQ1t73RqJlLCRxTwUhMp3OMMB0=";
+
+  doCheck = false;
```

Comment:
Can you add a comment to explain why this is disabled?

Reply:
Copy-paste error as well, fixed.